sys/log/log_printfnoformat/log_printfnoformat.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2015 Ludwig Ortmann <ludwig.ortmann@fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+/**
+ * @ingroup     core_log
+ * @{
+ *
+ * @file
+ * @brief       logging implementation that only prints the first argument
+ *
+ * @author      Ludwig Ortmann <ludwig.ortmann@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <stdarg.h>
+
+#include "log.h"
+
+void log_info(const char *format, ...)
+{
+    putchar('I');
+    putchar(':');
+    putchar(' ');
+    puts(format);
```

Comment:
Okay… I was under the impression, that `va_list` would contain some kind of (type identifier, value) tuples, but after reading the doc I reallize that this is obviously not the case :-/